samples/bluetooth/peripheral_hids_keyboard/README.rst

```diff
@@ -1,24 +1,145 @@
+
+This sample is similar to Zephyr's :ref:`zephyr:ble_peripheral` sample, except that it specifically exposes the HID GATT Service.
+It uses a report map for a generic keyboard.
 
 
 Requirements
 ************
 
-* A board with BLE support
+* One of the following development boards:
+
+  * nRF9160 DK board (PCA10090)
+  * nRF52840 Development Kit board (PCA10056)
+  * nRF52 Development Kit board (PCA10040)
+  * nRF51 Development Kit board (PCA10028)
+
+User interface
+**************
+
+Button 1:
+   Send one character of the predefined input ("hello\\n") to the computer.
+
+Button 2:
+   Simulate the Shift key.
+
+LED 1:
+   Indicates if Caps Lock is enabled.
+
+
 
-Building and Running
+Building and running
 ********************
 
 This sample can be found under :file:`samples/bluetooth/peripheral_hids_keyboard` in the |NCS| folder structure.
 
 See :ref:`gs_programming` for information about how to build and program the application.
+
+Testing
+=======
+
+After programming the sample to your board, you can test it either by connecting the board as keyboard device to a Microsoft Windows computer or by connecting to it with `nRF Connect for Desktop`_.
+
+Testing with a Microsoft Windows computer
+-----------------------------------------
+
+To test with a Microsoft Windows computer that has a Bluetooth radio, complete the following steps:
+
+1. Power on you development board.
+#. On your Windows computer, search for Bluetooth devices and connect to the device named "NCS HIDS keyboard".
+#. Open a text editor (for example, Notepad).
+#. Repeatedly push Button 1 on the board.
+   Every button press sends one character of the test message "hello" (the test message includes a carriage return) to the computer, and this will be displayed in the text editor.
+#. Push Button 2 and hold it while pushing Button 1.
+   Observe that the next letter of the "hello" message appears as capital letter.
+   This is because Button 2 simulates the Shift key.
+#. Turn Caps Lock on on the computer.
+   Observe that LED 1 turns on.
+   This confirms that the output report characteristic has been written successfully on the HID device.
+#. Turn Caps Lock off on the computer.
+   Observe that LED 1 turns off.
+#. Disconnect the computer from the device by removing the device from the computer's devices list.
+
+
+Testing with nRF Connect for Desktop
+------------------------------------
+
+To test with `nRF Connect for Desktop`_, complete the following steps:
+
+1. Power on you development board.
+#. Connect to the device from nRF Connect (the device is advertising as "NCS HIDS keyboard"), then bond to the device.
+   To bond, click the settings button for the device in nRF Connect, select **Pair**, check **Perform Bonding**, and click **Pair**.
+   Wait until the bond is established before you continue.
+   Observe that the services are shown in the connected device.
```

Comment:
Observe that the services of the connected device are shown.

I mean - they are not shown in the connected device but the content of the connected device is shown in nRF Connect.